ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.scss

```diff
@@ -20,7 +20,7 @@
     background: $primary-blue;
     color: white;
     height: 40px;
-    width: 100px;
+    width: 100px !important;
```

Comment:
instead of using important, it's best to increase specificity by including the parent class: 
```suggestion
  & &__dismiss-button {
    background: $primary-blue;
    color: white;
    height: 40px;
    width: 100px;
```
this results in the selector:
```css
.unconnected-account-alert .unconnected-account-alert__dismiss-button {
```
which is higher specificity than the .button class selector that these styles override.

Reply:
Unfortunately, I did try that and seems that the class `unconnected-account-alert` is never in use in [unconnected-account-alert](https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.js). This can work if I can wrap everything with the class in use? Is that desirable?

Reply:
Add the 'unconnected-account-alert' class to the popover in `unconnected-account-alert.js` as class name. this will make the suggestion work. sorry about that I thought i tested it but my working directory wasn't clean so no telling what happened 

```js
    <Popover
      title={isExtensionUrl(origin) ? t('currentExtension') : new URL(origin).host}
      subtitle={t('currentAccountNotConnected')}
      onClose={onClose}
      className="unconnected-account-alert"
      contentClassName="unconnected-account-alert__content"
      footerClassName="unconnected-account-alert__footer"
      footer={footer}
    >
```

![image](https://user-images.githubusercontent.com/4448075/93518056-ecb03880-f8f1-11ea-94c4-6f51ebd75b9b.png)

Reply:
0ad7475